src/Helpers/MiscHelper.js

```diff
@@ -273,3 +273,7 @@ export const translateUrlSortParameter = (value) => {
 
     return { key, direction };
 };
+
+export const groupsOrWorkspaces = (isWorkspaceEnabled) =>
+    isWorkspaceEnabled ? 'Workspaces' : 'Groups';
```

Comment:
This function only returns plural 'Workspaces' or 'Groups'. In most, if not all, of the uses in this app, we were passing in 'Group'.

![image](https://github.com/RedHatInsights/vulnerability-ui/assets/15373136/18131d87-8dce-40ad-ab3a-10f7675b4c23)

Reply:
Thank you for the review.  I will update the PR accordingly, ask another review!